microsoft-edge/webview2/release-notes.md

```diff
@@ -19,6 +19,69 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime or SDK specific.  
 
+## 1.0.955-prerelease
+
+Release Date: July 26, 2021  
+
+[NuGet package][NuGetGallery1.0.955-prerelease] \| Minimum Microsoft Edge version to load: 86.0.616.0 or newer \| Full API Compatibility: 93.0.967.0 or newer  
+
+### General  
+*   General WebView2 performance improvements.
+*   Added some ETW tracing support for WebView2.
+*   Removed Microsoft branding from edge://history.
+*   WebView2 has a new default Download UI.
+
+#### Experimental Features  
+*   Added [OpenTaskManagerWindow API][Webview2ReferenceWin32Icorewebview2experimental4ViewWebview210955PrereleaseOpentaskmanagerwindow] to launch a webview2 browser task manager.
+*   Added [NewWindowRequestedEventArgs][Webview2ReferenceWin32Icorewebview2experimentalnewwindowrequestedeventargsViewWebview210955PrereleaseGetName].
+*   Added [PDF Toolbar API][Webview2ReferenceWin32Icorewebview2experimentalnewwindowrequestedeventargsViewWebview210955PrereleaseGetName].
+
+#### Bug fixes  
+*   Fixed bug that broke the edge://downloads and edge://history pages. This change is Runtime-specific.
+*   Fixed bugs to improve reliability in the WebView2Loader.dll.
+*   Fixed bug in which NewWindowRequested event handler launches two window when handling links with target=_blank.
+*   Fixed a bug in WebView visual hosting would present a black flicker before startup.
+*   Fixed bug when add_WebResourceRequested was not working for WebView2 created via add_NewWindowRequested. \([\#616][GithubMicrosoftedgeWebviewfeedbackIssue616]\)
+*   Allowed the host app to set foreground on a different application in response to certain events (NavigationStarting, AddHostObjectToScript methods, WebMessageReceived, and NewWindowRequested). \([\#1092][GithubMicrosoftedgeWebviewfeedbackIssue1092]\)
+*   Fix bug to fire PermissionRequested event for Microphone. This change is Runtime-specific.\([\#1462][GithubMicrosoftedgeWebviewfeedbackIssue1462]\)
+*   Fixed bug when ExecuteScriptAsync was blocking forever after several successful runs. This change is Runtime-specific. \([\#1348][GithubMicrosoftedgeWebviewfeedbackIssue1348]\)
+*   Fixed bug that prevented non-ASCII filenames from being used for `ResultFilePath` in DownloadStartingEventArgs. \([\#1428][GithubMicrosoftedgeWebviewfeedbackIssue1428]\)
+*   Fixed bug where the default popup's title bar is not displayed entirely. This change is Runtime-specific. \([\#1016][GithubMicrosoftedgeWebviewfeedbackIssue1016]\)
+*   Added support for virtual host name mapping to work with Service Workers.
```

Comment:
This sounds like a new feature. Do we need a new section heading for that?

Reply:
changed to be in another section